src/app/map/suitability-map-panel/suitability-map-panel.component.html

```diff
@@ -22,14 +23,14 @@ <h2 class="map__control__title title">
                         <span class="heading__icon">
                             <i class="crop-icon crop-icon--{{ data.slug }}"></i>
                         </span>
-                        <span class="heading__text">{{ data.name }}</span>
+                        <span class="heading__text">{{ 'SUITABILITY.' + data.name | translate}}</span>
```

Comment:
Add space after the translate pipe like this `{{ 'SUITABILITY.' + data.name | translate }}`